### ISSUE TITLE
There is no navigation link for lessons

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt#3453]
**Repro steps:**
• Navigate to https://makecode.microbit.org/docs
• Click link "Lessons"

**Expected Result:**
There should have navigation link for every page
**Actual Result:**
There is no navigation link for every lesson
![4](https://user-images.githubusercontent.com/23466737/32648716-1b97d982-c632-11e7-941a-0bfde77bf442.gif)

**Comment by microsoftsam** [comment]
This is by design, Lessons are only there for legacy, so we don't want to surface them from the Sidebar.

**Comment by samelhusseini** [comment]
Lessons not in summary, and therefore they don't include a breadcrumb. By design.
